src/components/pages/module-form/ModuleFormContainer.jsx

```diff
@@ -1,11 +1,67 @@
 import React from 'react';
+import { Form, Input, Modal } from 'antd';
+import { useResetFormOnCloseModal } from './useResetFormOnCloseModal';
+
+// ModuleFormContainer component
+// props come from CourseFormContainer
```

Comment:
Excellent use of comments here. I like this pattern as it quickly shows where props are coming from without having to trace the flow of data through the app.